<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>POS TO WEB SKUS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>971</width>
      <height>651</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="1">
      <widget class="QPushButton" name="gatherDataButton">
       <property name="text">
        <string>Gather Data</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="web1lineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="pos1lineEdit"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="web2lineEdit"/>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="save2MButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="posSrchButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="web3lineEdit"/>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="pos3lineEdit"/>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="saveSKUSButton">
       <property name="text">
        <string>Save sku2sku</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="webSrchButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="pos2lineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>POS SKU SEARCH</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>MERGED DATA</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>POS NAME SEARCH</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="bclineEdit"/>
     </item>
     <item row="2" column="0" colspan="6">
      <widget class="QTableView" name="posTable"/>
     </item>
     <item row="4" column="0" colspan="6">
      <widget class="QTableView" name="webTable"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="barCodeSrchButton">
       <property name="text">
        <string>Search Bar Code</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>WEB SEARCH</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="6">
      <widget class="QTableView" name="mergedTable"/>
     </item>
     <item row="6" column="0" colspan="6">
      <widget class="QTableView" name="previewTable"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
